### PR TITLE
 [FIX] web: kanban: transaction error on record dragged twice

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/src/views/kanban/kanban_model.js [hunk 1]
```diff
@@ -449,8 +461,6 @@ export class KanbanDynamicGroupList extends DynamicGroupList {
             this.model.notify();
```
Comment (aab-odoo) [inline addons/web/static/src/views/kanban/kanban_model.js:461]: only catch NoTransactionInProgress here?

**Review by aab-odoo — APPROVED** [review]
I didn't know we can d&d records in a non grouped kanban view... Did we really support that before wowl?

File: addons/web/static/src/views/kanban/kanban_model.js (hunk 1 above)
Comment (Polymorphe57) [reply addons/web/static/src/views/kanban/kanban_model.js:457]: for the part before commit (update,...)

Comment (Polymorphe57) [reply addons/web/static/src/views/kanban/kanban_model.js:457]: here both commit and abort must happen after start and we know that no other transaction for dataRecordId can have started and then be ended.

**Comment by aab-odoo** [comment]
robodoo r+

**Pull request closed without merging**